lib/api/test/unit/units/validateBody/getBodySchemaType.test.js

```diff
@@ -0,0 +1,48 @@
+const { assert } = require('chai');
+const mediaTyper = require('media-typer');
+const { getBodySchemaType } = require('../../../../units/validateBody');
+
+describe('getBodySchemaType', () => {
+  describe('when given non-string schema', () => {
+    // ...
+  });
+
+  describe('when given json schema', () => {
+    describe('that contains object', () => {
+      const res = getBodySchemaType('{ "foo": "bar" }');
+
+      it('returns no errors', () => {
+        assert.isNull(res[0]);
+      });
+
+      it('returns "application/schema+json" media type', () => {
+        assert.deepEqual(res[1], mediaTyper.parse('application/schema+json'));
+      });
+    });
+
+    describe('that contains array', () => {
```

Comment:
I think we could convey our intentions better. Now it seems objects and arrays are supported as JSON Schema. While that's technically true, I think we want to rather say that we don't check the contents, so anything will do as JSON Schema.

What about we merge the two tests ("that contains object", "that contains array") into one, which is titled "that contains JSON", and which will use `'[1, 2, 3]'` as the tested value so it is clear that anything, even invalid JSON Schema document, will pass this particular function just fine, that there are no additional checks, and that it is an expected behavior.

Reply:
I've changed the way it's asserted.